js/reducers/passageReducer.js

```diff
@@ -38,10 +39,16 @@ function passageReducer(state = initialState, action) {
     case PREVIOUS_VERSE:
       return assignToEmpty(state, verseByIndex(state.verseIndex - 1));
     case ENABLE_RECALL:
+      // don't attempt to parse the text if we're already in recall mode
+      if (state.isRecalling) {
+        return state;
+      }
+
       return assignToEmpty(state, {
         verseText: state.verseText.split(' ').map(function (word) {
           return word[0] + word.slice(1, word.length).replace(/\w/g, ' ');
-        }).join(' ')
+        }).join(' '),
+        isRecalling: true
```

Comment:
Since all of the other actions are running through ```verseByIndex``` to set state, it's being reset to false when they're called.